### ISSUE TITLE
Field path does not support reference back to base table

### ISSUE THREAD

**Issue by dsenalik** [tripal/tripal#1767]
### Tripal Version

4.alpha2

### General Description

This is showing up for the sequence coordinates field. For this field, we have a gene feature which has sequence coordinates on another feature, e.g. chromosome as stored in the `featureloc` table.
If we want the name of this chromosome, we need to retrieve the source feature. In SQL this would be
`SELECT SF.uniquename FROM feature F LEFT JOIN featureloc FL ON F.feature_id=FL.feature_id LEFT JOIN feature SF ON FL.srcfeature_id=SF.feature_id`
For a field definition this would be 
`"path" => "feature.feature_id>featureloc.feature_id;featureloc.srcfeature_id>feature.feature_id;uniquename"`
Note that we have two different `feature.feature_id` in this path! Without some sort of alias mechanism we end up just getting the name of the gene, rather than of the chromosome.

### Steps to reproduce

To test, change Sukha's PR #1700 in `ChadoSequenceCoordinatesDefault.php` at line 104
```
//    $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, 'uniquename', $ft_uniqname_term, [
//      'action' => 'store',
//      'path' => 'feature.uniquename',
//    ]);
    $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, 'uniquename', $ft_uniqname_term, [
      'action' => 'store',
      'path' => 'feature.feature_id>featureloc.feature_id;featureloc.srcfeature_id>feature.feature_id;uniquename',
    ]);
```
and build a new docker. Create organism, then analysis, then load the citrus gff from https://tripal.readthedocs.io/en/latest/user_guide/example_genomics/genomes_genes.html then publish `Gene`

### Screenshots

Here the gene name is displayed instead of the scaffold name
![20240123_sequence-coordinates-table](https://github.com/tripal/tripal/assets/8419404/e70e28e8-25a6-46bb-a07f-6541b409625b)

![20240125_path_settings](https://github.com/tripal/tripal/assets/8419404/6cfb8c77-98d8-40ff-b6d6-91f863862250)

**Comment by dsenalik** [comment]
I tried this
```
     $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, 'uniquename', $ft_uniqname_term, [
       'action' => 'store',
       'path' => 'feature.feature_id>featureloc.feature_id;featureloc.srcfeature_id>srcfeature.feature_id;uniquename',
       'table_alias_mapping' => ['srcfeature' => 'feature'],
     ]);
```
but this error results
`ChadoRecords::initTable(). Invalid attempt to initalize a table as the base table cannot have an alias. The element tries to set a table alias, which is not supported: Array ( [base_table] => feature [chado_table] => feature [chado_column] => uniquename [table_alias] => srcfeature [column_alias] => gene_sequence_coordinates__uniquename [delta] => 0 [value] => [operation] => = [delete_if_empty] => [empty_value] => [field_name] => gene_sequence_coordinates [property_key] => uniquename ) `

**Comment by laceysanderson** [comment]
Should be fixed by PR #1788 🎉
